engines/agi/systemui.cpp

```diff
@@ -734,10 +759,19 @@ void SystemUI::drawSavedGameSlotSelector(bool active) {
 
 	_text->charAttrib_Push();
 	_text->charAttrib_Set(0, 15);
+	int16 column;
+	const char *arrow;
+	if (!_vm->isLanguageRTL()) {
+		column = 0;
+		arrow = "\x1a";
+	} else {
+		column = 33;
```

Comment:
`FONT_COLUMN_CHARACTERS - 7`? What's this value?

Reply:
Again, it's based on other magic number.